src/actions/resetPassword.js

```diff
@@ -0,0 +1,9 @@
+import axios from '../utils/axiosInstance';
+import { PASSWORD_RESET } from './types';
+
+const resetPassword = formdata => async (dispatch) => {
+  const { data: { message } } = await axios.post('/users/resetpassword', formdata);
```

Comment:
`async/await` should be wrapped in try-catch block to catch/handle exceptions.

Reply:
Resolved.